src/buildblock/ProjData.cxx

```diff
@@ -422,4 +422,44 @@ write_to_file(const string& output_filename) const
 
 }
 
+void
+ProjData::
+axpby(const float a, const ProjData& x,
+      const float b, const ProjData& y)
+{
+    int n = get_max_segment_num();
+    int nx = x.get_max_segment_num();
+    int ny = y.get_max_segment_num();
```

Comment:
What do you mean? There's no guarantee that `nx==ny` as far as I can see.

Reply:
no, but you do have to guarantee it by checking first. what would `apxby` mean with different sized objects?

Reply:
Didn't think much about it as I copied from SIRF. Anyway, in the new PR I use the `==` operator from `ProjDataInfo`, so that should solve it, correct?